graph.cpp

```diff
@@ -104,6 +145,11 @@ vector<set<Node>> Graph::connected_components() {
   GraphVerifier verifier {cum_in};
 #endif
 
+  // force DSU structure into memory
+  // if this call fails, there is no problem since unlock won't throw any
+  // errors if we try to unlock un-locked memory
+  mlock(parent, num_nodes*sizeof(Node));
```

Comment:
same

Reply:
Oops thought I removed this. I'll do that now.